SpeckleDynamo/pkg.json

```diff
@@ -2,7 +2,7 @@
   "license": "MIT",
   "file_hash": null,
   "name": "Speckle for Dynamo",
-  "version": "0.0.3",
+  "version": "0.0.4",
```

Comment:
In the future let's also update these for coherence, and let's find a way to automate it :):

https://github.com/speckleworks/SpeckleDynamo/blob/master/SpeckleDynamo/Properties/AssemblyInfo.cs#L54
https://github.com/speckleworks/SpeckleDynamo/blob/master/SpeckleDynamo/pkg.json#L18